data/free worlds start.txt

```diff
@@ -1615,6 +1780,124 @@ mission "FW Pirates 4.1"
+			`	A third senator speaks up, a woman whose name card identifies her as Sara, from Skymoot. "That will be all, Captain <last>. The Senate will confer and decide whether we want your continued services."`
+			`	You are led back out to the lobby, where you wait for hours. Finally, JJ comes and leads you to  Senator Sara's office. `
+			`	The senator gestures for you both to sit down and pours you a drink. "Captain <last>, you've put us in a bind. News of your actions is spreading like wildfire, and much of the public is firmly behind you and your actions. Many of us personally feel great admiration for your actions on Greenrock. Obviously, we can't condemn someone who has freed so many slaves and destroyed such a serious threat. From a practical standpoint, we are almost certain to need you and your ship in the coming battles with the Republic.`
+			`	Senator Sara sips her drink, then continues. "However, we are a government, and governments work on the rule of law. Your actions were morally correct, but legally, you've made a hell of a mess for us. Our government is still working out basic principles and the right balance of power. We cannot just allow ship captains in our forces to simply do as they please. Therefore you must have some kind of public reprimand and punishment.`
+			`	"Officially, we have decided that you will be suspended for your disobedience without pay. Separately, and rather unofficially, we have adjusted our clandestine operations budget to pay you a regular retainer. We will need your services again, and we hope you will come quickly when we contact you."`
+			goto goodbye
+			label goodbye
+			`	JJ walks you out of the Senate building. "I'm so sorry for the way I treated you back on Longjump," he says. "When we thought a pirate armada was ready to strike, we were willing to do anything to break them up, even if it meant making a very public example of you." His face reddens. "It took the shock of seeing my sister again to make me see sense. I hope you'll forgive me for it someday." You shake hands, and feel him pass you a folded was of paper. Then JJ is gone, striding off to another committee meeting. 
+			Back onboard <ship>, you open the paper. It reads,`
+			`	<first>,
+			`		DESTROY THIS NOTE AFTER YOU READ IT. We've been reviewing your recording of the battle over Greenrock. One of the ships you defeated had technology and fighters in it that were radically different from anything we've seen. We suspect that may have been a cell of Alphas. They've been known to capture fertile women in order to breed, so they had a vested interest in protecting Greenrock's slave market. We've heard rumors of mysterious hostile vessels raiding the eastern Syndicate; they may be connected. We don't have the resources to check any of this out, and I can't officially order you to do anything while you're suspended, but I suggest you check out these Korath Raiders and see if you can find a way to explore beyond human space. Godspeed.`
```

Comment:
How would he know that they are called Korath raiders?
How would he know that they are called the Korath in the first place?